testing/ncids-css-testing/.backstop/engine-scripts/playwright/clickAndHoverHelper.js

```diff
@@ -1,37 +1,49 @@
 module.exports = async (page, scenario) => {
   const hoverSelector = scenario.hoverSelectors || scenario.hoverSelector;
   const clickSelector = scenario.clickSelectors || scenario.clickSelector;
+	const focusSelector = scenario.focusSelectors || scenario.focusSelector;
```

Comment:
how did this get past the linter?

Reply:
It did not as it is not under src. :(. I prettier'd it up.